server/sonar-web/src/main/js/apps/background-tasks/components/Stats.js

```diff
@@ -65,10 +66,11 @@ export default class Stats extends React.PureComponent {
           {this.props.isSystemAdmin && (
             <Tooltip overlay={translate('background_tasks.cancel_all_tasks')}>
               <a
-                className="js-cancel-pending icon-delete spacer-left"
+                className="js-cancel-pending spacer-left link-no-underline"
                 href="#"
-                onClick={this.handleCancelAllPending}
-              />
+                onClick={this.handleCancelAllPending}>
+                <DeleteIcon className="text-text-top" />
```

Comment:
I wonder if we should avoid using this ? Because it doesn't has the same alignement in Firefox and Chrome... I feel like every time I use it I end up receiving feedback from a Firefox user that there is a bad alignement.

Reply:
It's fixed in my upcoming PR :)